pkgs/development/python-modules/bayespy/default.nix

```diff
@@ -15,12 +15,25 @@ buildPythonPackage rec {
     sha256 = "ed0057dc22bd392df4b3bba23536117e1b2866e3201b12c5a37428d23421a5ba";
   };
 
-  checkInputs = [ pytest nose glibcLocales ];
+  patches = [
+    # Change from scipy to locally defined epsilon
+    # https://github.com/bayespy/bayespy/pull/126
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/bayespy/bayespy/pull/126.patch";
```

Comment:
In the case of a PR that's been merged it's usually best to use the merge-commit from the PR. In this case https://github.com/bayespy/bayespy/commit/9be53bada763e19c2b6086731a6aa542ad33aad0.patch would be the url to use. In theory PRs can be mutated whereas commits should be immutable.

It's also nice to give patches names, in this case I would use `name = "locally-defined-epsilon.patch";`.